frontend/src/common/top_bar.tsx

```diff
@@ -32,7 +32,10 @@ export function TopBar({ children, className }: { children: React.ReactNode; cla
     </div>
   );
 }
-export const TopBarTitle = primitiveWithClassname('h2', 'text-xl font-bold');
+export const TopBarTitle = primitiveWithClassname(
+  'h2',
+  'text-xl font-bold text-nowrap text-ellipsis overflow-hidden',
```

Comment:
isnt `text-nowrap text-ellipsis overflow-hidden` the same as `truncate` https://tailwindcss.com/docs/text-overflow?